core/src/main/kotlin/io/github/mstachniuk/graphqlschemafromintrospectiongenerator/internal/GeneratorImpl.kt

```diff
@@ -34,23 +34,29 @@ class GeneratorImpl {
                 .joinToString(" | ")
     }
 
-    private fun printSchema(response: IntrospectionResponse): String {
+    private fun printSchema(response: IntrospectionResponse): String =
         if (isCustomQueryType(response) || isCustomMutationType(response) || isCustomSubscriptionType(response)
         ) {
-            var output = "schema {\n"
+            "schema {\n" +
             if (isCustomQueryType(response)) {
-                output = "$output${margin}query: ${response.data.schema.queryType?.name}\n"
-            }
+                "${margin}query: ${response.data.schema.queryType?.name}\n"
+            } else {
```

Comment:
Thank you for the PR. Generally, I like all the changes, outside this one (the repeated `else { "" }` block. Maybe StringBuilder will be better here?

Reply:
That was more an example how to do that without variables using if expression.
Actually (actually) it also showed no tests for else branch.
But - yes surely there should be a better way to write this part of code...

Reply:
But as for StringBuilder I am not allowed to use StringBuilder directly :-)